R/importNGWMN_wml2.R

```diff
@@ -161,9 +161,9 @@ importWaterML2 <- function(input, asDateTime=FALSE, tz="UTC") {
                       time = character(0), dateTime = character(0), value = numeric(0),
                       uom = character(0), comment = character(0), stringsAsFactors = FALSE))
   }
-  rawTime <- xml_text(xml_find_all(TVP,".//wml2:time"))
+  rawTime <- xml_text(xml_find_all(returnedDoc, "./wml2:point/wml2:MeasurementTVP/wml2:time"))
```

Comment:
So why was the old way not working? can we change that xpath on line 158? If not, should we delete that line? What was nice about using the ".//" if I remember correctly was that some WaterML2 had slightly different wrappers than others....(like, feature collections vs wml2 collections vs. single timeseries).

Reply:
Did we decide to give up on collections when we moved to `xml2`?

Reply:
So, we could still use the old style xpath.  The problem was using `xml_find_all` on `TVP` (a nodeset) instead of `returnedDoc` (a node).  

Since we are searching the tree from higher up now, I thought it would be safer to specify the path in case there was a node with the same name inside the overall metadata section as in the time-value pairs, that could result in a column length mismatch.

Reply:
What do you mean by collections?

Reply:
When there is more than one site for example and/or more than one paremeter. Take Rhode Island (because it's small):
https://waterservices.usgs.gov/nwis/dv/?format=waterml,2.0&stateCd=ri&siteStatus=all

I think we decided that the WaterML2 parser should be pretty generic....and we were focus on essentially a TVP parser. But I'm not sure I remember exactly